README.md

```diff
@@ -751,3 +751,32 @@ Availability:
 -datetime.timezone.utc
 +datetime.UTC
 ```
+
+### Replace unnecessary lambdas in `collections.defaultdict` calls
```

Comment:
Wasn't sure where to put the documentation section so I placed it at the end, let me know if you prefer me to move it somewhere else. (Maybe so exemple could be trimmed down too)

Reply:
I would maybe put it after the dict / set comprehension stuff since it's ~related-ish